### PR TITLE
Adding templates for various conditional logic operators.

### PR DESCRIPTION
**Pull request by flyinbutrs**
#### New Template Items:
* fn_or
* fn_and
* fn_if
* fn_not
* equal
* not_equal

### CODE REVIEW

File: lib/cloudformation-ruby-dsl/cfntemplate.rb [hunk 1]
```diff
@@ -391,6 +391,30 @@ def join(delim, *list)
 # Variant of join that matches the native CFN syntax.
 def join_list(delim, list) { :'Fn::Join' => [ delim, list ] } end
 
+def equal(one, two) { :'Fn::Equals' => [one, two] } end
+
+def fn_not(item) { :'Fn::Not' => [] } end
```
Comment (jonaf) [inline lib/cloudformation-ruby-dsl/cfntemplate.rb]: `item` is not used in the body of this method.

File: lib/cloudformation-ruby-dsl/cfntemplate.rb [hunk 2]
```diff
@@ -391,6 +391,30 @@ def join(delim, *list)
...
+
+def fn_or(*list)
```
Comment (jonaf) [inline lib/cloudformation-ruby-dsl/cfntemplate.rb]: Instead of `list`, can we call this `condition_list` or `conditions`? Ditto for `fn_and`.

**Comment by flyinbutrs** [comment]
Sure, I will make those changes and commit again.

**Comment by jonaf** [comment]
Would you mind terribly squashing your commits down to just one? It will make getting this released much easier.

Jona

On May 18, 2015, at 7:41 PM, Daniel Rosenbloom <notifications@github.com<mailto:notifications@github.com>> wrote:


Sure, I will make those changes and commit again.

-

**Pull request closed without merging**